launcher/sdw_updater_gui/UpdaterApp.py

```diff
@@ -185,17 +185,17 @@ def run(self):
 
         # apply dom0 state
         self.progress_signal.emit(10)
-        result = Updater.apply_dom0_state()
         # add to results dict, if it fails it will show error message
-        results["apply_dom0"] = result.value
+        results["apply_dom0"] = Updater.apply_dom0_state()
 
         self.progress_signal.emit(15)
         # rerun full config if dom0 checks determined it's required,
         # otherwise proceed with per-VM package updates
         if Updater.migration_is_required():
             # Progress bar will freeze for ~15m during full state run
             self.progress_signal.emit(35)
-            Updater.run_full_install()
+            # add to results dict, if it fails it will show error message
+            results["apply_all"] = Updater.run_full_install()
```

Comment:
it looks like `run_full_install` does not return `UpdateStatus.UPDATES_FAILED` or error codes like it looks like you expect here. you might want to modify `run_full_install` to have exception handling similar to `apply_dom0_state`.

Reply:
I'm not sure I follow, those modifications are made as part of this PR (see changes above to `Updater.py`)

Reply:
I'm sorry, I must have been looking at a different branch. I see your changes now.